kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/engine/spec/interceptor/SpecFinishedInterceptor.kt

```diff
@@ -19,7 +19,10 @@ internal class SpecFinishedInterceptor(private val listener: TestEngineListener)
    ): Result<Map<TestCase, TestResult>> {
       val start = timeInMillis()
       val value = fn(ref)
-      val duration = (timeInMillis() - start).milliseconds
+
+      // We deliberately use the deprecated method here to maintain backwards compatibility with Kotlin 1.5.
+      // See https://github.com/kotest/kotest/issues/3059.
+      val duration = Duration.milliseconds(timeInMillis() - start)
```

Comment:
Kotest 5.x is 1.6+ only, so we don't need to worry about < 1.6

Reply:
This fixes the issue described in https://github.com/kotest/kotest/issues/3059, which allows Kotest to be used to test Gradle plugins (Gradle uses Kotlin 1.5). Because the test I've added for the Gradle plugin itself uses Kotest, this was required. 

If I rewrite the tests to use something other than Kotest, then this change isn't required.

Reply:
Ok I see, this is fine then.